src/borg/logger.py

```diff
@@ -165,24 +165,38 @@ def setLevel(self, *args, **kw):
             return self.__logger.setLevel(*args, **kw)
 
         def log(self, *args, **kw):
+            if 'msgid' in kw:
+                kw.setdefault('extra', {})['msgid'] = kw.pop('msgid')
```

Comment:
either is pretty simple... just that for f(kw) I'd have to go back and use git-rebase... ;)